warehouse-management-service/db/migrations/000001_create_warehouse_table.up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS warehouse (
+    id TEXT PRIMARY KEY DEFAULT gen_random_uuid (),
+    name TEXT,
+    latitude TEXT,
+    longitude TEXT
```

Comment:
There are more appropriate types for lat, long values. Some options you can look at:
- [Points](https://www.postgresql.org/docs/current/datatype-geometric.html#id-1.5.7.16.5) (built-in type)
- [Geometry](https://postgis.net/docs/geometry.html), [Geography](https://postgis.net/docs/geography.html) (from the [PostGIS](https://postgis.net/) extension)

I'd guess PostGIS is overkill at this stage, but points should work well.

Reply:
Used Points here instead.